ts/src/base/Exchange.ts

```diff
@@ -3322,6 +3322,19 @@ export default class Exchange {
         throw new NotSupported (this.id + ' cancelOrder() is not supported yet');
     }
 
+    async cancelOrders (ids: string[], symbol: string = undefined, params = {}): Promise<any> {
```

Comment:
@pcriadoperez Usually even when we emulate a unified method we still perform a single REST call which is not the case here so I'm not sure yet if we should open this precedent or not, for instance should we also emulate `fetchOrders` with multiple `fetchOrder` calls?

@kroitor what do you think?

Reply:
I'd rather not emulate the missing functionality (cause that's a lot of emulation to add across the lib). I would probably leave the rate limiting control one the user side for cases like these.